.idea/py42.iml

```diff
@@ -4,6 +4,7 @@
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
     </content>
+    <orderEntry type="jdk" jdkName="Python 3.7 (py3)" jdkType="Python SDK" />
```

Comment:
can you undo this change and ignore this file? there is a `.gitignore` file in the .idea folder.